extensions/azure-extensions/src/main/java/io/druid/storage/azure/CompressionUtilsWrapper.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Druid - a distributed column store.
+ *  Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.druid.storage.azure;
+
+import com.google.common.base.Predicate;
+import com.google.common.io.ByteSource;
+import com.metamx.common.CompressionUtils;
+import com.metamx.common.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CompressionUtilsWrapper
+{
+
+  public FileUtils.FileCopyResult unzip(
```

Comment:
Only way I could find to add unit tests for AzureDataSegmentPuller once all CompressionUtils methods are static and it make life harder on unit tests.

The only other option that I saw was adding [PowerMock](https://code.google.com/p/powermock/) that allows me to mock static and final classes/methods, but to be frank, I don't like much this idea.

Anyways, I can remove it and also remove the Puller unit test.

Any thoughts?